processor/src/test/java/org/mapstruct/ap/testutil/runner/CompilingStatement.java

```diff
@@ -102,12 +102,7 @@ protected CompilationCache initialValue() {
         }
     };
 
-    private static final List<String> LIBRARIES = Arrays.asList(
```

Comment:
Btw. `AnnotationNotFoundTest` fails for me on this branch. Does it pass for you? I guess it is somehow related to this change here.